app/controllers/api/v1/invoices/inv_invoice_items_controller.rb

```diff
@@ -0,0 +1,8 @@
+class Api::V1::Invoices::InvInvoiceItemsController < ApplicationController
```

Comment:
I know it will look repetitive, but similar to my other PR comment: can we fully name invoice?